utils/bluez-tools/files/bt-agent.init

```diff
@@ -0,0 +1,37 @@
+#!/bin/sh /etc/rc.common
```

Comment:
Please use procd for this service init and remove all comments

Reply:
done